### PR TITLE
Add translation for submission server errors

### PR DESCRIPTION
**Pull request by melloskitten**
## Description
Hi everybody! This PR adds missing texts for Submission Errors. The corresponding JIRA issue can be found [here](https://jira.itc.sap.com/browse/EXPOSUREAPP-1825). Thanks for checking.

### CODE REVIEW

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -482,6 +482,11 @@ Sie können die Hintergrundaktualisierung für die App jederzeit in den Gerätee
 "ExposureSubmissionResult_RegistrationDate" = "Registriert am";
 
 /* Exposure Submission */
+
+"ExposureSubmissionError_InvalidPayloadOrHeaders" = "Invalide Payload oder Header erhalten.";
+
+"ExposureSubmissionError_RequestCouldNotBeBuilt" = "Die Submission Request konnte nicht korrekt gebaut werden.";
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:488]: are those the official translations?

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -482,6 +482,11 @@ You can switch on background fetching for the app at any time in the device sett
 "ExposureSubmissionResult_RegistrationDate" = "Registered on ";
 
 /* Exposure Submission */
+
+"ExposureSubmissionError_InvalidPayloadOrHeaders" = "Received an invalid Payload or headers.";
+
+"ExposureSubmissionError_RequestCouldNotBeBuilt" = "The Submission Request could not be built correctly.";
+
```
Comment (johannesrohwer) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:489]: capitalisation is odd here.

**Review by johannesrohwer — DISMISSED** [review]
thanks for the tests! I just have two tiny questions regarding the texts.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (melloskitten) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:488]: Nope.  I will ask @kaluju whether there are some english/german translations for these error messages.

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (melloskitten) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:489]: agreed, will check with @kaluju

**Comment by melloskitten** [comment]
@johannesrohwer and @inf2381, I have now added the texts for the alerts as recommended by @janetback.
She has proposed to prefix the technical error descriptions (which should be in English, regardless of localization) with a localized generic data transmission error text. Could you please re-review? Thanks & cheers! 😊

**Review by inf2381 — APPROVED** [review]

**Review by ChristianKienle — APPROVED** [review]

**Review by janetback — CHANGES_REQUESTED** [review]
Hi, the German text is fine. I don't quite understand why this PR contaims changes in other language files? The other language file should receive their changes from translation only. Perhaps this is why the English version contains an error ("ocurred" instead of "occurred").
In my opinion DE files only should be changed by dev. Please check, thanks, Janet

**Comment by melloskitten** [comment]
@janetback I am a bit confused. We have developers that don't speak German. In the previous process, @kaluju has told me that I should add keys into all other languages and just fill them with the english placeholder value. Did this process change?

**Comment by melloskitten** [comment]
This is completely new information to me. I will adjust it accordingly.

**Comment by melloskitten** [comment]
hi @janetback, could you double check whether the PR is fine now? thanks so much.

**Review by janetback — APPROVED** [review]

**Pull request merged**